### ISSUE TITLE
Is this project still going?

### ISSUE THREAD

**Issue by VolleyballCALVA** [TheChed/PSXMSFS#5]
Sorry to use the issue  function, I'd like to ask is this project still going? I really love your work. The WidePSX is not very stable on the pushback function since when I tried, after the pushback complete, the aircraft in MSFS jumped back to where PSX at (in the gate, before pushback). Hope this is still in development. Thank you very much.

Best Regards,
Liu

**Comment by TheChed** [comment]
Hi Liu,

Yes still going on. Latest version should address your issues

**Comment by VolleyballCALVA** [comment]
That's a great news! Thank you very much.

**Comment by VolleyballCALVA** [comment]
Hello, sorry to bother, but since I got no reply in the Aerowinx forum, I came here for help. It's weird that I got no EICAS messages with the V1.1 release, Am I done something wrong? I downloaded the PSXMSFS.exe and put in a folder with the simconnect.dll, then I started MSFS, then PSX, then use cmd to start the PSXMSFS.exe after I went into the cockpit of both sims. The "P" key function act normally without the EICAS message. Thank you in advanced.

**Comment by TheChed** [comment]
Hi,

I think I disabled the EICAS message. But pressing the P key ion MSFS should toggle PSX between slave/master

**Comment by VolleyballCALVA** [comment]
Hello~ It seems like the project is still going?  Thank you very much for the hard work!

**Comment by TheChed** [comment]
Yes still going on.... but slower pace.  But still open to new ideas to add to the project

**Comment by VolleyballCALVA** [comment]
Great! Thank you! The most wanted feature is the switchable master/slave (which is already done), and the stable connection between MSFS and Aerowinx. I still randomly get the disconnection in flight and really hope this can be fixed. Thank you very much for your great work.
May I ask if I could get a copy of current development version for testing if I still got disconnection? Thanks in advanced.
